imports/plugins/included/email-notifications/server/i18n.js

```diff
@@ -0,0 +1,83 @@
+import i18next from "i18next";
+import i18nextSprintfPostProcessor from "i18next-sprintf-postprocessor";
+import { Hooks, Logger } from "/server/api";
+import { fetchTranslationResources } from "/lib/api/i18n";
+import { Packages } from "/lib/collections";
+
+/**
+ * Hooks to setup core i18n imports during Reaction init
+ */
+Hooks.Events.add("onCoreInit", () => {
+
+  // init i18next for SSR translations
+  i18next.
+    use(i18nextSprintfPostProcessor).
+    init({
+      debug: false,
+      defaultNS: "core",
+      resources: fetchTranslationResources()
+    }, (err) => {
+      if (err) throw new Meteor.Error("No translations resources found.", err);
+      Logger.debug("Finishing loading of server side translations.");
+    });
+});
+
+Hooks.Events.add("afterCoreInit", () => {
+  function getSource(templateId) {
+    // using layout where in the future a more comprehensive rule based
+    // filter of the email templates can be implemented.
+    const exists = Packages.findOne({
+      "layout.template": templateId
+    });
+    if (exists) {
+      let lang = "en";
+      const shopLocale = Meteor.call("shop/getLocale");
+
+      if (shopLocale && shopLocale.locale && shopLocale.locale.languages) {
+        lang = shopLocale.locale.languages;
+      }
+      tplSource = Templates.findOne({
+        template: templateId,
+        language: lang
+      });
+      if (tplSource.source) {
+        return tplSource.source;
+      }
+    }
+
+    const path = Npm.require("path");
```

Comment:
This should just be `require('path');` with an `import path from "path";` at the top